### PR TITLE
Add @githubteacher bio

### PR DESCRIPTION
**Pull request by githubteacher**
This adds a bio for @githubteacher as requested in #8.

Hey @crichID, what do you think of this bio?  I would :heart: your thoughts on this.

### CODE REVIEW

**Comment by crichID** [comment]
@githubteacher maybe we should add something about being an octocat

**Comment by githubteacher** [comment]
I think this is a :sparkles: idea @crichID!  Let's find a place to swap something out.

File: bios/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## GitHub Teacher
+- Trainer at GitHub
+- Loves training
```
Comment (githubteacher) [inline bios/githubteacher.md]: How about we replace this with the part about being an Octocat, @crichID?

Comment (crichID) [inline bios/githubteacher.md]: I agree this is the perfect place for it :sparkles:

**Comment by githubteacher** [comment]
@crichID I think this is good to :ship:!  :+1:

**Pull request merged**